.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish and Tag
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
Let's create the branch 'main' and activate continuous_delivery when merged to 'main'

Reply:
I did this change on the yml.
Once the PR is accepted, I will open the branch, push develop into it, and tag it.